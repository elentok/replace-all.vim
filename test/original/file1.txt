I am jim1
I am joe1
I am bob1
I am bob9
