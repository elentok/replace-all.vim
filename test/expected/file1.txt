I am jim1
I am joe1
I am sheldon1 
I am bob9
